### PR TITLE
SYMBIAN: Add more generators for project files.

### PR DESCRIPTION
**Pull request by fedor4ever**
Add handler for command line options.
Fix for generators for project files.
Remove function duplicate: Common::KeymapperDefaultBindings *OSystem_SDL_Symbian::getKeymapperDefaultBindings().
Update help.

### CODE REVIEW

**Comment by fedor4ever** [comment]
On hold till next commit.

File: backends/platform/symbian/src/SymbianOS.cpp [hunk 1]
```diff
@@ -92,6 +89,15 @@ void OSystem_SDL_Symbian::initBackend() {
 	if (!ConfMan.hasKey("savepath")) {
 		ConfMan.set("savepath", savePath);
 	}
+	
+#if !RELEASE_BUILD
+	_LIT(KDefaultBetaExtraPath,"!:\\DATA\\ScummVM\\BETA\\");
```
Comment (sev-) [inline backends/platform/symbian/src/SymbianOS.cpp:94]: Is it supposed to be in this hardcoded directory?

File: backends/platform/symbian/symbian_builder/scummvm_hrh.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+def Generate_ScummVm_hrh(build):
+   uids = get_UIDs(build)
+   defines = ""
+   for i in range(len(uids)):
+      if i > 1:
+         defines += macro_templatte %(i, uids[i])
+   data = hrh_template %(uids[0], defines)
+   SafeWriteFile(os.path.join("src", "ScummVm.hrh"), data)
+
+
```
Comment (sev-) [inline backends/platform/symbian/symbian_builder/scummvm_hrh.py:58]: End line is missing at the end of the file

**Review by sev- — CHANGES_REQUESTED** [review]
I did a cursory check and almost every line in the C++ code does not conform to our coding standards.

Without fixes, this could not be accepted in-tree and I did not spend time looking at the actual functionality seeing so many formatting issues

File: backends/platform/symbian/src/SymbianOS.cpp (hunk 1 above)
Comment (fedor4ever) [reply backends/platform/symbian/src/SymbianOS.cpp:94]: Yes. First letter - '!' placeholder for real drive name. Drive name can be from A to Y. In that folder placed all datafiles for beta versions by installer.
Currently datafiles doesn't fit in phone memory.

**Comment by fedor4ever** [comment]
This code still not ready.

**Pull request closed without merging**